openedx/core/djangoapps/profile_images/tests/test_views.py

```diff
@@ -219,6 +221,57 @@ def test_upload_self(self, mock_make_image_version, mock_log):  # pylint: disabl
 
         self.check_upload_event_emitted(old=TEST_UPLOAD_DT, new=TEST_UPLOAD_DT2)
 
+    @ddt.data(
+        ('image/jpeg', '.jpg'),
+        ('image/jpeg', '.jpeg'),
+        ('image/pjpeg', '.jpg'),
+        ('image/pjpeg', '.jpeg'),
+        ('image/png', '.png'),
+        ('image/gif', '.gif'),
+        ('image/gif', '.GIF'),
```

Comment:
Though it would be testing existing functionality, I like the idea of extending this `ddt` data to also cover `content-type=multipart/form-data`, since there's no other test that I see here which tests all the mimetypes with that upload method.